src/Pages/OfficerDB/Edits/Adder.js

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react'
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Input
+} from 'reactstrap'
+import ConfirmModule from './ConfirmationModule'
+import Uploader, { upload, getFileName, checkUpload } from './Uploader'
+
+const enums = require('../../../Enums')
+const api = require('../../../APIFunctions/OfficerDB')
+
+export default function Adder (props) {
+  const [personDropdown, setPersonDropdown] = useState(false)
+  const [roleDropdown, setRoleDropdown] = useState(false)
+  const [person, setPerson] = useState('')
+  const [role, setRole] = useState('')
+  const [linkedIn, setLinkedIn] = useState('')
+  const [github, setGithub] = useState('')
+  const [quote, setQuote] = useState('')
+  const [queryUsers, setQueryUsers] = useState('')
+  const optionalInputs = [
+    {
+      placeholder: 'linkedIn',
+      onChange: event => setLinkedIn(event.target.value)
+    },
+    {
+      placeholder: 'Github',
+      onChange: event => setGithub(event.target.value)
+    },
+    {
+      placeholder: 'Quote',
+      onChange: event => setQuote(event.target.value)
+    }
+  ]
+
+  function findUser (query) {
+    query.toLowerCase().trim()
+      ? setQueryUsers(
+        props.users.filter(user =>
+          user.firstName.toLowerCase().includes(query)
+        )
+      )
+      : setQueryUsers('')
+  }
+
+  async function submitOfficer () {
```

Comment:
this function should go inside `APIFunctions/`

Reply:
Above comment